### ISSUE TITLE
Should have Recheck button on About dialog

### ISSUE THREAD

**Issue by akarasulu** [subutai-io/tray#140]
Seems this checks once then does not update. Nice to have a recheck button activate once first check is done. This is a **VERY** minor enhancement suggestion. Obviously many other things are more important.

**Comment by Lezh1k** [comment]
There are p2p, tray, rh check frequency options In "Settings" dialog. Here we can specify time interval. If update is available - about dialog will be updated. But sure we need button "Refresh" .

**Comment by Lezh1k** [comment]
done
